src/sha256.ts

```diff
@@ -0,0 +1,5 @@
+import { sha256 as rawSha256 } from "@noble/hashes/sha256"
+import type { Input } from "@noble/hashes/utils"
+import { Effect } from "effect"
```

Comment:
In libs, you always want to import like this:

```ts
import  * as Either from "effect/Either"
```

Otherwise, this can break tree-shaking in many cases.